code/modules/projectiles/guns/ballistic.dm

```diff
@@ -221,7 +221,7 @@
 		name = "sawn-off [src.name]"
 		desc = sawn_desc
 		w_class = WEIGHT_CLASS_NORMAL
-		weapon_weight = WEAPON_MEDIUM // Nope. Not unless you want broken wrist.
+		weapon_weight = WEAPON_MEDIUM // FUCK IT WE BALL
```

Comment:
change it back, less it's reminds a reminder of you not balling

Reply:
I must remind myself of this sin that I bear upon my flesh.